arch/arm/soc/atmel_sam/sam4s/soc.c

```diff
@@ -184,7 +172,9 @@ static ALWAYS_INLINE void clock_init(void)
 	while (!(PMC->PMC_SR & PMC_SR_LOCKA))
 		;
 
-	/* Setup PLLB. */
+	/*
+	 * Setup PLLB.
```

Comment:
PLLB is used by USB only. Until we provide an USB driver we should not touch this PLL. It only increases power consumption.